buildpacks/nodejs-function-invoker/src/main.rs

```diff
@@ -69,6 +69,7 @@ impl Buildpack for NodeJsInvokerBuildpack {
             .unwrap_or_else(|| DetectResultBuilder::fail().build())
     }
 
+    #[allow(deprecated)]
```

Comment:
Is this addition intentional, or a rebase issue?

```suggestion
```

Reply:
Rebase issue that, for whatever reason, only happened with this PR and none of the other. This `#[allow(deprecated)]` annotation is now removed (again).